plugins/station.py

```diff
@@ -156,3 +200,69 @@ def station(bot, mask, target, args):
     except KeyError:
         bot.privmsg(target, "Error while retrieving traffic data.")
         raise Exception()
+
+
+@cron('23 0 * * *')
+@asyncio.coroutine
+def cccongress_update_cron(bot):
```

Comment:
That definition name seems a little too much copypasta? Should probably have a more appropriate name for the plugin..?